exercises/high-scores/runtests.jl

```diff
@@ -0,0 +1,38 @@
+# canonical data version 4.0.0

```

Comment:
There are no tests for `scores` when `sorted` is given as an argument. Please add some or remove the argument.